q2galaxy/core/usage.py

```diff
@@ -55,8 +55,12 @@ def _make_params(self, action, input_opts):
 
         for case in signature_to_galaxy(sig, input_opts):
             test_xml = case.tests_xml()
-            if test_xml is not None:
-                self.xml.append(case.tests_xml())
+            if type(test_xml) is not list:
+                test_xml = [test_xml]
+
+            for test_node in test_xml:
+                if test_node is not None:
```

Comment:
Was there a reason `None` would show up in these lists? that sounds like a misbehaving handler and the default error that would come out of it would be fine. I don't think we need this guard.

Reply:
Ah, disregard I see why now. It might be easier to do a `continue` at the top instead